static/Functions.js

```diff
@@ -208,7 +208,7 @@ function processAttachments(isUpload, cb) {
           url: config.documentServiceUrl + "/api/Attachment",
           type: 'POST',
           contentType: "application/json; charset=utf-8",          
-          data: data,          
+          data: JSON.stringify(data),          
```

Comment:
Right... :)

Reply:
It works now ;-)

Reply:
Cool! (Why are you working today?!)
This is what happen when I just take a quick look at my emails on the weekend... Sorry I drugged you into this... :)

Reply:
np - very bad weather today ;-)

Reply:
ok. enjoy the rest of your weekend! :)
I see there's a small confilict, I'll resolve and merge this thing tomorrow...

Reply:
no hurry, the conflict comes from the change of the user api from POST to GET - should be straight forward - enjoy the weekend